src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/resource/ReportResource.kt

```diff
@@ -537,6 +538,63 @@ class ReportResource(
     }
   }
 
+  @PostMapping("/{id}/description-addendums")
+  @PreAuthorize("hasRole('ROLE_MAINTAIN_INCIDENT_REPORTS') and hasAuthority('SCOPE_write')")
+  @ResponseStatus(HttpStatus.CREATED)
+  @Operation(
+    summary = "Appends to the report description",
+    description = "Requires role MAINTAIN_INCIDENT_REPORTS and write scope. " +
+      "Authentication token must provide a username which is recorded as the report’s modifier.",
+    responses = [
+      ApiResponse(
+        responseCode = "201",
+        description = "Returns updated incident report",
+      ),
+      ApiResponse(
+        responseCode = "400",
+        description = "Invalid request",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      ),
+      ApiResponse(
+        responseCode = "401",
+        description = "Unauthorized to access this endpoint",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      ),
+      ApiResponse(
+        responseCode = "403",
+        description = "Missing required role. Requires the MAINTAIN_INCIDENT_REPORTS role with write scope.",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      ),
+      ApiResponse(
+        responseCode = "404",
+        description = "Data not found",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      ),
+    ],
+  )
+  fun addDescriptionAddendum(
+    @Schema(
+      description = "The internal ID of the report",
+      example = "11111111-2222-3333-4444-555555555555",
+      requiredMode = Schema.RequiredMode.REQUIRED,
+    )
+    @PathVariable
+    id: UUID,
+    @RequestBody
+    @Valid
+    request: DescriptionAddendum,
```

Comment:
Normally we don't use the same object for the request as the DTO return value.  The request object might be different

Reply:
e..g AddendumRequest instead